File: integration-tests/java-17/src/main/java/io/quarkus/it/hibernate/panache/person/Person.java
package io.quarkus.it.hibernate.panache.person;

import jakarta.persistence.Entity;

import io.quarkus.hibernate.orm.panache.PanacheEntity;

@Entity
public class Person extends PanacheEntity {
    public String firstname;
    public String lastname;
    public Status status = Status.ALIVE;
}

File: integration-tests/mongodb-panache-kotlin/src/main/kotlin/io/quarkus/it/mongodb/panache/person/Person.kt
package io.quarkus.it.mongodb.panache.person

import org.bson.codecs.pojo.annotations.BsonId

class Person {
    @BsonId
    var id: Long? = null
    var firstname: String? = null
    var lastname: String? = null
    var status: Status = Status.ALIVE
}